src/main/java/cs361/battleships/models/Game.java

```diff
@@ -4,14 +4,19 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Random;
 
 import static cs361.battleships.models.AtackStatus.*;
 
 public class Game {
 
-    @JsonProperty private Board playersBoard = new Board();
-    @JsonProperty private Board opponentsBoard = new Board();
```

Comment:
This was fine as it was. In Java, you can initialize in the member declaration instead of using a default constructor.